Birthday-2024-Project/ScenePrefabs/PuzzleUiManager.tscn

```diff
@@ -31,6 +35,19 @@ anchor_bottom = 1.0
 grow_horizontal = 2
 grow_vertical = 2
 
+[node name="HBoxContainer2" type="HBoxContainer" parent="MainScreen/MarginContainer"]
```

Comment:
make this container the same size as Utility Buttons from Main Screen for consistency

Reply:
Will try to implement, but Godot's UI sometimes... 😅I'm still learning too